charts/rancher-monitoring/100.2.0+up40.1.2/app-README.md

```diff
@@ -14,6 +14,21 @@ The chart installs the following components:
 
 For more information, review the Helm README of this chart.
 
+## Upgrading from 100.1.0+up19.0.3 to 100.2.0+up40.1.2
+
+### Rebased the chart to upstream version 40.1.2. 
+### Noticeable changes:
+Grafana: 
+- switched from AngularJS to React
+
+Prometheus-Operator:
+- introduced a change to avoid name collisions with AlertmanagerConfig (https://github.com/prometheus-operator/prometheus-operator/commit/80a72c42f5c62012d27330b7c5203fd110d7f764)
+
+Node-exporter: 
```

Comment:
this is handled by an upgrade job now:

https://github.com/rancher/charts/blob/6aef72f6ce1b80ac702a2c9da17cf544f48cf1b3/charts/rancher-monitoring/100.2.0%2Bup40.1.2/files/upgrade/scripts/delete-workloads-with-old-labels.sh#L7